security/rbac.adoc

```diff
@@ -0,0 +1,24 @@
+[[security-rbac]]
+= Roles and Access Management
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+Roles and Access Management in a Cluster Federation
+
+In July of 2016, Kubernetes 1.3 introduced Kubernetes Federated Clusters.  Federation is useful for very large scale, global deployments. Multiple clusters can be useful for high-availability across multiple availability zones or to enable common management of deployments or migrations across multiple cloud providers, such as AWS, Google Cloud, and Azure.
```

Comment:
Not sure I like this here, as https://github.com/openshift/origin/  states: 

Feature | Stage (version) | Description
------------ | ------------- |  -------------
Federated Clusters | Alpha (1.3), Beta (1.4, 1.5),	Tech Preview (1.3, 1.4, 1.5) | A Kubernetes federation server may be used against Origin clusters with the appropriate credentials today. Known issues include tenant support in federation and the ability to have consistent access control between federation and normal clusters. **No Origin specific binary is being distributed for federation at this time.**